index.bs

```diff
@@ -134,11 +139,11 @@ interface PerformanceElementTiming : PerformanceEntry {
 A {{PerformanceElementTiming}} object reports timing information about one associated image.
 
 Each {{PerformanceElementTiming}} object has these associated concepts, all of which are set on creation:
-* A <dfn>request</dfn> containing the image request.
-* A <dfn>response</dfn> containing the image response.
+* A <dfn>request</dfn> containing the image request (if the entry is for image content).
```

Comment:
what happens with these values for text entries? Might be better to set it to null or somesuch

Reply:
I had set them to null when creating text, moved it to default to null.